src/app/signup/trainer/_layout.tsx

```diff
@@ -0,0 +1,30 @@
+import { PropsWithChildren } from "react";
+import { Flex, Space, Typography } from "antd";
+type TrainerSingUpLayoutProps = {
+  currentStep: number;
+} & PropsWithChildren;
+
+const TrainerSignUpLayout = ({ children, currentStep }: TrainerSingUpLayoutProps) => {
+  const comments: string[] = ["간단한 기본 정보를 입력해주세요 :)", "상세 정보를 입력해주세요 :)"];
```

Comment:
```suggestion
  const comments = ["간단한 기본 정보를 입력해주세요 :)", "상세 정보를 입력해주세요 :)"];
```

어차피 추론될 것 같아서 따로 타입 지정 안해주어도 될 것 같아용